datasetinsights/datasets/groceries_real.py

```diff
@@ -4,20 +4,15 @@
 import logging
 import os
 import shutil
-import zipfile
 from pathlib import Path
 
-import tensorflow as tf
-from google.protobuf import text_format
 from PIL import Image
 
 import datasetinsights.constants as const
 from datasetinsights.io.bbox import BBox2D
-from datasetinsights.io.gcs import GCSClient
 
 from .base import Dataset
 from .exceptions import DatasetNotFoundError
-from .protos import string_int_label_map_pb2
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Why keeping GroceriesReal(Dataset) class here?

Reply:
Although we don't need this class in general, I think it's the best place to keep it, cause it belongs to the dataset category.

Reply:
It does not have tf and torch dependencies so I have not removed.